package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```

Comment:
What's the purpose of adding this file?

Reply:
Acutely not sure must have been added by a tool like a package manager, perhaps added when I updated azdev. I removed id.